corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -790,9 +790,12 @@ def _record_timing(self, status):
 
         tags['type'] = 'sync' if self.params.sync_log_id else 'restore'
 
-        if settings.ENTERPRISE_MODE and self.params.app and self.params.app.copy_of:
-            app_name = slugify(self.params.app.name)
-            tags['app'] = '{}-{}'.format(app_name, self.params.app.version)
+        if settings.ENTERPRISE_MODE:
+            if self.params.app and self.params.app.copy_of:
+                app_name = slugify(self.params.app.name)
+                tags['app'] = '{}-{}'.format(app_name, self.params.app.version)
+            else:
+                tags['app'] = ''
```

Comment:
Does it also need to be set if `settings.ENTERPRISE_MODE` is false?

Reply:
no, as long as the setting doesn't change at runtime